Jenkinsfile

```diff
@@ -22,23 +22,75 @@ properties([
 ])
 
 void setGitHubBuildStatus(String context, String message, String state) {
-  step([
-    $class: 'GitHubCommitStatusSetter',
-    reposSource: [$class: 'ManuallyEnteredRepositorySource', url: 'https://github.com/nuxeo/nuxeo-elements'],
-    contextSource: [$class: 'ManuallyEnteredCommitContextSource', context: context],
-    statusResultSource: [$class: 'ConditionalStatusResultSource', results: [[$class: 'AnyBuildResult', message: message, state: state]]],
-  ])
+  if (env.DRY_RUN != "true") {
+    step([
+      $class: 'GitHubCommitStatusSetter',
+      reposSource: [$class: 'ManuallyEnteredRepositorySource', url: 'https://github.com/nuxeo/nuxeo-elements'],
+      contextSource: [$class: 'ManuallyEnteredCommitContextSource', context: context],
+      statusResultSource: [$class: 'ConditionalStatusResultSource', results: [[$class: 'AnyBuildResult', message: message, state: state]]],
+    ])
+  }
+}
+
+def isPullRequest() {
+  return BRANCH_NAME =~ /PR-.*/
 }
 
+def getPackageVersion() {
+  container('nodejs') {
+    return sh(script: 'npx -c \'echo "$npm_package_version"\'', returnStdout: true).trim()
+  }
+}
+
+def getReleaseVersion() {
+  def preid = 'rc'
```

Comment:
What pre-release ID should we be using? 'rc' is pretty common, but 'alpha' and 'beta' are also used, and I've seen 'pre' as well. Polymer uses 'rc' for their library and 'pre' for individual elements.

Reply:
I'd say we stick to "rc" for now since that's what these versions are anyway "release candidates"